ui/webui/src/components/users/Accounts.jsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect } from "react";
+
+import {
+    Form,
+    FormGroup,
+    Stack,
+    StackItem,
+    TextInput,
+    Title,
+} from "@patternfly/react-core";
+
+import "./Accounts.scss";
+
+const _ = cockpit.gettext;
+
+const CreateAccount = ({
+    idPrefix,
+}) => {
+    const [fullName, setFullName] = useState();
+    const [userAccount, setUserAccount] = useState();
+    const [password, setPassword] = useState();
+    const [confirmPassword, setConfirmPassword] = useState();
+
+    return (
+        <div id={idPrefix + "-create-account"}>
+            <Stack hasGutter>
```

Comment:
Instead of using Stack/StackItems you can put everything in a `<Form isHorizontal>`, spacings will be correct automatically.

Reply:
I am using the Stack to get the spacing between rows (like in the mockup), without it it looks like this:
![Screenshot from 2023-10-30 17-18-18](https://github.com/rhinstaller/anaconda/assets/1220237/d27dd0da-0c9b-4b94-9365-f8615c7f9e34)

Reply:
The proper way to space form elements it to wrap everything with a '<Form isHorizontal>' and let Patternfly do the trick :)

Reply:
Ah, this works if I move Form from `Accounts` to `Create Account` (having it in Accounts does not work). I wanted the Form to contain also (future) root component. I guess I'll use two forms then - `Create Account` and the other one for the root component? Well, it can be addressed when I add the UI for root. For now I'll move Form to `Create Accounts`.